test/solidity/Facets/AcrossFacetV3.t.sol

```diff
@@ -119,6 +121,134 @@ contract AcrossFacetV3Test is TestBaseFacet {
         }
     }
 
+    function test_canSwapAndBridgeTokensWithOutputAmountPercent()
+        public
+        assertBalanceChange(
+            ADDRESS_DAI,
+            USER_SENDER,
+            -int256(swapData[0].fromAmount)
+        )
+        assertBalanceChange(ADDRESS_DAI, USER_RECEIVER, 0)
+        assertBalanceChange(ADDRESS_USDC, USER_SENDER, 0)
+        assertBalanceChange(ADDRESS_USDC, USER_RECEIVER, 0)
+    {
+        vm.startPrank(USER_SENDER);
+
+        // prepare bridgeData
+        bridgeData.hasSourceSwaps = true;
+
+        // reset swap data
+        setDefaultSwapDataSingleDAItoUSDC();
+
+        // Set output amount percent to 85%
+        validAcrossData.outputAmountPercent = uint64(850000000000000000); // 85.00%
+        validAcrossData.outputAmount = 10000; // This will be ignored
+
+        // approval
+        dai.approve(_facetTestContractAddress, swapData[0].fromAmount);
+
+        //prepare check for events
+        vm.expectEmit(true, true, true, true, _facetTestContractAddress);
+        emit AssetSwapped(
+            bridgeData.transactionId,
+            ADDRESS_UNISWAP,
+            ADDRESS_DAI,
+            ADDRESS_USDC,
+            swapData[0].fromAmount,
+            bridgeData.minAmount,
+            block.timestamp
+        );
+
+        vm.expectEmit(true, true, true, true, _facetTestContractAddress);
+        emit LiFiTransferStarted(bridgeData);
+
```

Comment:
We should also add an check for the `V3FundsDeposited` event exposed by across to verify that the calculated amount matches our expectations.